bot.ts

```diff
@@ -95,7 +95,11 @@ Here are the sports grounds on our campus:
 [BHU Pool](https://goo.gl/maps/uxAmEWXRPqcyMRfR7)
 `;
 
-const bot = new Bot(Deno.env.get('TELEGRAM_BOT_TOKEN') || '');
+const bot = new Bot('5673881243:AAEsIDrhX34JzPw76DJqu13wAFtstM33T00');
```

Comment:
```suggestion
const bot = new Bot(Deno.env.get('TELEGRAM_BOT_TOKEN') || '');
```

Can't expose the bot token in the `bot.ts` file.